### DISCUSSION TITLE
Why UID and GID are different between alpine and ubuntu

### DISCUSSION THREAD

**Discussion by carlos-m-rodrigues** [docker-library/rabbitmq#642] [General]
Hello guys,

Why UID and GID are different between alpine and ubuntu? In case of alpine the user id is 100 and group id is 101, in ubuntu the user id and group id is 999. 
If we are mapping volumes to the container this could be an issue if we try to use the same image with different flavors.

**Comment by tianon** [comment]
The difference is a somewhat unfortunate historical artifact at this point -- when we initially created these, we didn't set an explicit number, so later we _did_ but decided to set it to what it'd been being set to automatically to minimize the amount of breakage caused by a change. :disappointed:
